sql/index/pilosa/index.go

```diff
@@ -1,13 +1,22 @@
 package pilosa
 
 import (
+	"crypto/sha1"
+
 	errors "gopkg.in/src-d/go-errors.v1"
 
 	pilosa "github.com/pilosa/go-pilosa"
 	"gopkg.in/src-d/go-mysql-server.v0/sql"
 	"gopkg.in/src-d/go-mysql-server.v0/sql/index"
 )
 
+type expressionHash []byte
```

Comment:
I think, we totally don't need it. We can totally really on expressions as strings